runtime/src/native_loader.rs

```diff
@@ -81,12 +81,12 @@ impl NativeLoader {
     }
 
     #[cfg(windows)]
-    fn library_open(path: &PathBuf) -> std::io::Result<Library> {
+    fn library_open(path: &PathBuf) -> Result<Library, libloading::Error> {
```

Comment:
@jackcmay (Ignore bunch of other code in this pr); This file's 2 hunks (this and the next) is the only smart-contract-related code changes to bump `libloading` to follow upstream's new API. I believe this changes don't have problems. I'm asking you just in case. :)

Reply:
@ryoqun Looks good to me!